jinahub/indexers/searcher/FaissSearcher/config.yml

```diff
@@ -6,4 +6,6 @@ metas:
     - faiss_searcher.py
   name: faiss_searcher
   workspace: workspace/
+runtime_args:
+  name: faiss_searcher
```

Comment:
Is it necessary? Where does `name` is used?

Reply:
It's checked in a recent fix in master. But that fix required a subsequent fix, that wasn't merged at the time. Will try with latest core master